docs/content/common-subs/required-packages.md

```diff
@@ -32,33 +41,72 @@
         ```
         ``` title="kind - https://kind.sigs.k8s.io/docs/user/quick-start/"
-        curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.14.0/kind-linux-amd64 && chmod +x ./kind && mv ./kind /usr/local/bin
+        # For AMD64 / x86_64
+        [ $(uname -m) = x86_64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.14.0/kind-linux-amd64
+        # For ARM64
+        [ $(uname -m) = aarch64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.14.0/kind-linux-arm64 
+        chmod +x ./kind && sudo mv ./kind /usr/local/bin/kind
         ```
         ``` title="kubectl - https://kubernetes.io/docs/tasks/tools/ (version range expected: 1.23-1.25)"
-        curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x kubectl && mv ./kubectl /usr/local/bin/kubectl
+        # For AMD64 / x86_64
+        [ $(uname -m) = x86_64 ] && curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && chmod +x kubectl && mv ./kubectl /usr/local/bin/kubectl
+        # for ARM64 / aarch64
+        [ $(uname -m) = aarch64 ] && curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubectl" && chmod +x kubectl && mv ./kubectl /usr/local/bin/kubectl
+        ```
+        ``` title="ko - https://github.com/ko-build/ko (required for compiling KubeStellar Syncer)"
+        VERSION="0.14.1" # choose the latest version (without v prefix)
+        OS="Linux"    # or Darwin
+        # set proper architecture ( ARCH=x86_64  # or arm64, i386, s390x)
+        # for AMD64 / x86_64
+        [ $(uname -m) = x86_64 ] && ARCH="x86_64"
+        # for ARM64 / aarch64
+        [ $(uname -m) = aarch64 ] && ARCH="arm64"
+        # (simplified install without running slsa-verifier)
+        curl -sSfL "https://github.com/ko-build/ko/releases/download/v${VERSION}/ko_${VERSION}_${OS}_${ARCH}.tar.gz" > ko.tar.gz
+        tar xzf ko.tar.gz ko
+        chmod +x ./ko && sudo mv ./ko /usr/local/bin/ko
+        ```
+        [GO v1.19](https://gist.github.com/jniltinho/8758e15a9ef80a189fce) - You will need GO to compile and run kcp and the KubeStellar processes.  Currently kcp requires go version 1.19.
+    === "Windows"
+        ``` title="Chocolatey - https://chocolatey.org/install#individual"
+        Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
         ```
-        [GO v1.19](https://gist.github.com/jniltinho/8758e15a9ef80a189fce) - You will need GO to compile and run kcp and the KubeStellar scheduler.  Currently kcp requires go version 1.19.
-    === "WSL"
         ``` title="jq - https://stedolan.github.io/jq/download/"
         choco install jq -y
         choco install curl -y
```

Comment:
Let's not bury installation of `curl` under the heading of `jq`.

Reply:
That is the Windows Subsystem for Linux chunk of the required_packages.md file; I was focusing on getting the Fedora-family section fixed in this PR.
And then going back to check the others for another PR, especially since by experimenting I realized the sections for installing **ko** were incomplete in all versions.